app/src/main/java/ch/hikemate/app/model/route/HikeRoutesRepositoryOverpass.kt

```diff
@@ -389,10 +387,14 @@ class HikeRoutesRepositoryOverpass(private val client: OkHttpClient) : HikeRoute
 
       // If the description is not set, we'll set it to the from - to, if available
       // We also assert that the name is not the same as the description
-      if (description == null && from != null && to != null && finalName != "$from - $to") {
+      if (description == null && from != null && to != null && selectedName != "$from - $to") {
         description = "$from - $to"
       }
 
+      // Remove fix me from the name
+      val finalName =
+          selectedName?.replace(Regex("""(\? - fixme|fixme - | - fixme| fixme|fixme |[?])"""), "")
+
```

Comment:
Question: Nice regex! Have you done some research about other "unwanted names" ?

Reply:
Since it's defined by user themselves, the only research I've been able to do is by looking around on the maps for names that are incorrects